skyportal/model_util.py

```diff
@@ -11,6 +11,7 @@
     'Annotate',
     'Manage users',
     'Manage sources',
+    'Manage sources publishing',
```

Comment:
Maybe manage sources is enough? A question for @mcoughlin. That or the upload data permission idk.

Reply:
Maybe we stick with Manage Sources for now, and reconsider after seeing how it works in production.

Reply:
@antoine-le-calloch tagging you so you see that. You can remove that new permission and use Manage sources everywhere you used `Manage sources publishing`